independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/handlers/RestInitialHandler.java

```diff
@@ -37,8 +46,13 @@ public RestInitialHandler(Deployment deployment) {
             }
             initialChain[initialChain.length - 1] = this;
         }
-        this.requestContext = deployment.getThreadSetupAction();
-        this.requestContextFactory = deployment.getRequestContextFactory();
+        if (blockingDefault == BlockingDefault.BLOCKING) {
```

Comment:
the `requestContextFactory` class already has a method called `isDefaultBlocking` which seems to be meant for this purpose. 
Perhaps, it's all about to properly implement the VertxRequestContextFactory.isDefaultBlocking to take account the `@Blocking` annotation from the application main class.

Reply:
It is kind of similar, however the point of that is to convey whether or not the Resource Method is to be blocking or not.

Reply:
I updated the variable name to make things more clear